meshtastic/mesh_interface_runtime/send_pipeline.py

```
@@ -0,0 +1,854 @@
+        with self._node_db_lock:
+            my_node_num = self.myInfo.my_node_num if self.myInfo is not None else None
+
+        if my_node_num is not None and destinationId != my_node_num:
+            self._interface._wait_connected()
+
+        toRadio = mesh_pb2.ToRadio()
+
+        nodeNum: int = 0
+        if destinationId is None:
+            raise self._interface.MeshInterfaceError("destinationId must not be None")
+        elif isinstance(destinationId, int):
```

Ruff (0.15.7)

[warning] Unnecessary `elif` after `raise` statement

Remove unnecessary `elif`

(RET506)